ddp-workspace/projects/ddp-sdk/src/lib/components/languageSelector.component.ts

```diff
@@ -19,25 +19,25 @@ import { Subscription } from "rxjs";
         <mat-icon class="ddp-dropdown-arrow">arrow_drop_down</mat-icon>
       </button>
       
-      <mat-menu #menu="matMenu">
+      <mat-menu #menu="matMenu" class="language-menu">
         <button mat-menu-item *ngFor="let lang of getUnselectedLanguages()" (click)="changeLanguage(lang)">{{lang.displayName}}</button>
       </mat-menu>
     </div>
   `
 })
 export class LanguageSelectorComponent implements OnInit, OnDestroy {
   @Input() isScrolled: boolean;
+  @Output() visibility: EventEmitter<boolean> = new EventEmitter();
```

Comment:
This is smart--I hadn't thought about cases where a second language could be added to a study later on, but this is obviously useful for that case.

Reply:
Could we rename to `visible` or `isVisible`? Would be more indicative of a boolean value.

Reply:
6b3113d8d097c846e2613265c5bba53e25c1ff80